### DISCUSSION TITLE
Welcome to METASPACE Discussions!

### DISCUSSION THREAD

**Discussion by lmacielvieira** [metaspace2020/metaspace#1561] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Keep yourself updated at the `#announcements` channel
  * Ask questions you’re wondering about and find the answers to your questions at the `#Q&A` channel.
  * Share ideas and new feature requests for METASPACE  at the `#ideas` channel.
  * Remember that this is a community we build together 💪.

**Comment by UjjalkumarDas-create** [comment]
Hi, how can we upload large dataset from the Bruker's RapiFlex spatial lipidomic dataset? 

Thanks

**Reply by lmacielvieira** [reply]
Hi @UjjalkumarDas-create, could you send an email to contact@metaspace2020.org? I will provide further instructions there.
